Bundle/I18nBundle/Resources/config/services.yml

```diff
@@ -26,11 +26,9 @@ services:
             - '@victoire_i18n.locale_resolver'
         tags:
             - { name: routing.loader }
-
-    victoire_i18n.kernelrequest.listener:
-        class: Victoire\Bundle\I18nBundle\Listener\KernelRequestListener
+    victoire_i18n.available_locales_extension:
+        class: Victoire\Bundle\I18nBundle\Extension\AvailableLocalesExtension
```

Comment:
this service should be private to allow Symfony to inline it (you never need to do a `$container->get('victoire_i18n.available_locales_extension')` when using it, as it is only meant to be injected in Twig)